lib/prometheus/client/histogram.rb

```diff
@@ -94,6 +94,16 @@ def values
         end
       end
 
+      def init_label_set(labels)
+        base_label_set = label_set_for(labels)
+
+        @store.synchronize do
+          (@buckets + ["+Inf", "sum"]).each do |bucket|
```

Comment:
Damn, sorry, this is my bad. I just realized we have an `attr_reader :buckets`
Could you change this `@buckets` to just `buckets`? :D

Reply:
It's all about consistency 😛